cassandra/src/main/scala/akka/stream/alpakka/cassandra/package.scala

```diff
@@ -0,0 +1,22 @@
+/*
+ * Copyright (C) 2016 Lightbend Inc. <http://www.lightbend.com>
+ */
+package akka.stream.alpakka.cassandra
+
+import com.google.common.util.concurrent.{ FutureCallback, Futures, ListenableFuture }
+
+import scala.concurrent.{ Future, Promise }
+
+package object cassandra {
```

Comment:
Good point. Thanks.

I had two issues,
* I had to change the object name from `cassandara`, I think the `private[cassandra]` declaration was referencing the object instead of the package, which stopped it working.
* Then I had to add a implicit def to convert to the opts. It seems like the implicit class wasnt getting picked up with the private.

I that right,